src/views/catalog/customize/components/PersistentVolumeClaimSelect/PersistentVolumeSelectProject.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+import { PersistentVolumeClaimModel } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  FormGroup,
+  Select,
+  SelectOption,
+  SelectVariant,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+
+import { filter } from './utils';
+
+type PersistentVolumeSelectProjectProps = {
+  projectsName: string[];
+  selectedProject: string;
+  onChange: (newProject: string) => void;
+};
+
+export const PersistentVolumeSelectProject: React.FC<PersistentVolumeSelectProjectProps> = ({
+  selectedProject,
+  projectsName,
+  onChange,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [isNamespacePVCOpen, setNamespaceOpen] = React.useState(false);
+
+  const onSelect = React.useCallback(
+    (event, selection) => {
+      onChange(selection);
+      setNamespaceOpen(false);
+    },
+    [onChange],
+  );
+
+  const fieldId = 'pvc-project-select';
+
+  return (
+    <FormGroup
+      label={t('Persistent Volume Claim project')}
+      fieldId={fieldId}
+      id={fieldId}
+      isRequired
+      className="pvc-selection-formgroup"
+    >
+      <Select
```

Comment:
add `maxHeight={400}` to make the inner select scrollable.
Add it to all `Select` instances actually

Reply:
okay thanks